tox.ini

```
@@ -5,6 +5,7 @@ skip_missing_interpreters = true
 [testenv]
 deps = pytest==4.6.10
        pytest-cov==2.8.1
+       importlib_metadata
```

Only needed for < 3.8:
```suggestion
       importlib_metadata; python_version < '3.8'
```